numpy-mp/numpy-mp-demo-deployment.yaml

```diff
@@ -91,12 +91,12 @@ spec:
           limits:
             cpu: "10"
           requests:
-            cpu: "0.5"
+            cpu: "10"
```

Comment:
Set it to "10" then it implies this deploy can only run on the node which has more than 10 cpu cores. 
I think if you insist this setting, at least you have to add some comments in the README.md.

Reply:
changed it back to 0.5